app/views/Contact/Contact.js

```diff
@@ -1,17 +1,87 @@
 import React from "react";
-import Container from "widgets/shared/Container/Container";
-import { View, Text } from "react-native";
+import { Linking, Text , TextInput, View, TouchableOpacity  } from "react-native";
+import qs from 'qs';
 
+import Container from "widgets/shared/Container/Container";
+import { useForm } from "../../hooks/useForm";
 import { contactStyles } from "./ContactStyles";
 
+
+
 const Contact = () => {
+ 
+  const {to, subject, body, onChange} = useForm( {
+    to:'eresElMejor@ereselmejor.com',
+    subject:'',
+    body:''
+  })
+
   return (
-    <Container>
-      <View style={contactStyles.screenContainer}>
-        <Text></Text>
+    <Container style={contactStyles.container}>
+      
+      <View style={contactStyles.container}>
+        <Text style={contactStyles.textInput} > Haz tu consulta</Text>
+        <TextInput 
+          style={contactStyles.input}
+          placeholder="eresElMejor@ereselmejor.com"
+          editable={false}
+          value={to}
+          
+        />
+        <Text style={contactStyles.textInput} > Motivo de la consulta</Text>
+        <TextInput 
+          style={contactStyles.input}
+          placeholder="Indica un motivo"
+          onChangeText={ (value) => onChange(value, 'subject') }
+          value={subject}
+        />
+        <Text style={contactStyles.textInput} > Escribe tu consulta</Text>
+        <TextInput 
+          style={contactStyles.input}
+          placeholder="Escribe tu consulta"
+          onChangeText={ (value) => onChange(value, 'body') }
+          value={body}
+          
+        />
+        <TouchableOpacity
```

Comment:
Aquí se podría usar el componente que creó Lucho, CustonButton. Lo único que se tendría que modificar es que loe pudieses pasar al componente, por props tus estilos.

El código quedaría algo así:
`        <CustomButton
          title="Send Email"
          customStyles={contactStyles.button}
          action={() => {
            sendEmail(to, subject, body);
          }}
        />`

Y el componente del botón para que te permita pasarle la prop con los estilos:

`const CustomButton = ({
  title = "",
  action = null,
  children = null,
  customStyles,
}) => {
  return (
    <TouchableHighlight style={[styles.button, customStyles]} onPress={action}>
      <View style={styles.container}>
        {children}
        <Text style={{ ...styles.text, ...{ paddingLeft: children ? 10 : 0 } }}>
          {title}
        </Text>
      </View>
    </TouchableHighlight>
  );
};`